rn/Teacher/src/modules/speedgrader/components/Slider.js

```diff
@@ -239,6 +239,24 @@ export default class Slider extends Component<Props, State> {
     this.moveTo(value, saveScore)
   }
 
+  sliderAccessibilityLabel () {
+    if (this.props.excused) {
+      return i18n('Excused')
+    } else if (this.props.score !== null) {
+      return i18n(`{ score, number } out of { pointsPossible, number } {
+        points, plural,
+             =1 {point}
+          other {points}
+      } possible`, {
```

Comment:
`{pointsPossible, number} {points, plural}` seems like an odd choice, when they can be easily combined. Also the keyword `one` isn't the same thing as `=1`. Since we are mapping to the linguistic singular, `one` would be better here.

```
... out of { pointsPossible, plural,
    one {# point possible}
  other {# points possible}
}
```

Reply:
Ohh I forgot that you could do that. Great thanks!